README.md

```diff
@@ -109,8 +110,88 @@ MIT/Expat
+We next want to add the .allcontributors file:
+
+```bash
+$ tributors init allcontrib
+INFO:allcontrib:Generating .all-contributorsrc for datalad/datalad
+$ tributors update allcontrib
+INFO:allcontrib:Updating .all-contributorsrc
+INFO:allcontrib:Updating .tributors cache from .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor glalteva in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor adswa in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor chrhaeusler in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor soichih in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor mvdoc in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor mih in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor yarikoptic in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor loj in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor feilong in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor jhpoelen in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor andycon in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor nicholsn in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor adelavega in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor kskyten in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor TheChymera in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor effigies in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor jgors in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor debanjum in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor nellh in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor emdupre in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor aqw in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor vsoch in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor kyleam in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor driusan in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor overlake333 in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor akeshavan in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor jwodder in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor bpoldrack in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor yetanothertestuser in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor bhanuprasad14 in .all-contributorsrc
+WARNING:tributors:allcontrib does not support updating from orcids.
+WARNING:tributors:allcontrib does not support updating from email.
+```
+
+We can then populate the shared .tributors file:
+
+```bash
+$ tributors update-lookup allcontrib
+```
+
+And then we can rely on the [GitHub action](.github/workflows/update-contributors.yml) to update contributors, and
+look for new orcids as we did above.
```

Comment:
that action -- what would it do? (I think - submit PRs with suggested changes)
What to do if we need to skip some contributor(s)? (and thus reject a PR I guess)

Reply:
The action is what will (after merge) ping the GitHub api to find new contributors, update the .all-contributorsrc, and the badge / table rendered in the README.md.